src/CONST.ts

```diff
@@ -853,7 +853,7 @@ const CONST = {
     // It's copied here so that the same regex pattern can be used in form validations to be consistent with the server.
     VALIDATE_FOR_HTML_TAG_REGEX: /<([^>\s]+)(?:[^>]*?)>/g,
 
-    VALIDATE_FOR_LEADINGSPACES_HTML_TAG_REGEX: /<([\s]+[\s\w~!@#$%^&*(){}[\];':"`|?.,/\\+\-=<]+.*[\s]*)>/g,
+    VALIDATE_FOR_LEADINGSPACES_HTML_TAG_REGEX: /<([\s]+.+[\s]*)>/g,
```

Comment:
```suggestion
    VALIDATE_FOR_LEADINGSPACES_HTML_TAG_REGEX: /<([\s]+)(?:[^>]*?)>/g,
```
Maybe this regex is better as it will be closer to backend